### PR TITLE
Update serviceUriPrefix in values.yaml

### PR DESCRIPTION
**Pull request by dfitchett**
## What was the problem?

Vets-api and VRO service URI for Ep Merge App are mismatched. 

## How does this fix it?[^1]

Updates the service uri

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: helm/domain-ee-ep-merge-app/values.yaml
Comment (dfitchett) [inline helm/domain-ee-ep-merge-app/values.yaml:1]: This change makes the serviceUriPrefix the same as whats in the `helm/domain-ee-max-cfi-app/values.yaml`. Easier to make the change here than in vets-api.

**Review by nanotone — APPROVED** [review]
👍

**Review by agile-josiah — APPROVED** [review]
LGTM

**Review by msnwatson — APPROVED** [review]

**Comment by dfitchett** [comment]
Not changing.

**Pull request closed without merging**